src/containers/Map/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import styles from './styles.sass'
+import L from 'leaflet'
+
+export default class Map extends Component {
+  componentWillReceiveProps() {
```

Comment:
I have nothing against ESLint, but I don't thing this to be important feature. Please add it yourself if you like.